heron/healthmgr/src/java/com/twitter/heron/healthmgr/diagnosers/SlowInstanceDiagnoser.java

```diff
@@ -56,6 +59,7 @@ public Diagnosis diagnose(List<Symptom> symptoms) {
     ComponentMetrics pendingBufferMetrics = waitQDisparityComponents.get(bpMetrics.getName());
     if (pendingBufferMetrics == null) {
       // no wait Q disparity for the component with back pressure. There is no slow instance
+      LOG.info("no wait Q disparity for the component with back pressure.");
```

Comment:
This log message does not look very helpful. Either remove it or augment with wait q sizes and back pressure metrics.

Reply:
updated log message